tests/test-cases/Group6-VIC-Machine/6-02-Default.robot

```diff
@@ -23,3 +23,23 @@ Delete with defaults
     ${ret}=  Run  bin/vic-machine-linux delete --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD}
     Should Contain  ${ret}  vic-machine-linux delete failed:  resource pool
     Should Contain  ${ret}  /Resources/virtual-container-host' not found
+
+Wrong Password No Panic
+  Set Test Environment Variables
+  ${ret}=  Run  bin/vic-machine-linux create --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+  Should Contain  ${ret}  vic-machine-linux create failed
+
+  ${ret}=  Run  bin/vic-machine-linux delete --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+  Should Contain  ${ret}  vic-machine-linux delete failed
+
+  ${ret}=  Run  bin/vic-machine-linux inspect --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+  Should Contain  ${ret}  vic-machine-linux inspect  failed
+
+  ${ret}=  Run  bin/vic-machine-linux ls --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+  Should Contain  ${ret}  vic-machine-linux ls failed
+
+  ${ret}=  Run  bin/vic-machine-linux upgrade --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+  Should Contain  ${ret}  vic-machine-linux upgrade failed
+
+  ${ret}=  Run  bin/vic-machine-linux configure --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+  Should Contain  ${ret}  vic-machine-linux configure failed
```

Comment:
I'll add a step that checks for the word "panic" even though it's technically redundant it will make the test look more complete